docs/config/README.md

```diff
@@ -330,7 +330,7 @@ date is read from the `AWSUME_EXPIRATION` env var.
 
 | Option              | Default                                                           | Description                                                                                                 |
 | ------------------- | ----------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
-| `format`            | `'on [$symbol($profile )(\($region\) )(\[$duration\] )]($style)'` | The format for the module.                                                                                  |
+| `format`            | `"on [$symbol($profile )(\($region\) )(\[$duration\] )]($style)"` | The format for the module.                                                                                  |
```

Comment:
`'` is used over `"` in some cases to avoid needing to escape `\` as `\\`. Please avoid replacing `'` with `"`.

Reply:
Thank you very much. I see.I could modify it to `\\` with double quotes, but is it preferable to change it back to single quotes?

Reply:
I would prefer keeping the single quotes to keep it more readable.

Reply:
Corrected here. Please check it.
https://github.com/starship/starship/pull/4428/commits/2ef07e79b59dbc76704beec8aef45f503f344bb7